tests/bsim/bluetooth/host/adv/extended/tests_scripts/ext_adv_conn.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+# Copyright (c) 2024 Croxel, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+# Extended advertising test:
+#
+# - Connectable: In addition to broadcasting advertisements, it is connectable
+#   and restarts advertisements once disconnected. The scanner/central scans
+#   for the packets and establishes the connection, to then disconnect
+#   shortly-after.
+
+source ${ZEPHYR_BASE}/tests/bsim/sh_common.source
+
+simulation_id="ext_adv"
```

Comment:
you can't have both simulations share the same ID.
The runner runs them in parallel and bsim will get confused.

Reply:
Whoops - Sorry about that. It's been corrected now.